README.md

```diff
@@ -53,8 +50,7 @@ To contribute a patch, follow these instructions (note that installing
 ```
 
 
-License (see also LICENSE.txt)
-==============================
+## License (see also LICENSE.txt)
```

Comment:
Hi @IMS94 I haven't applied *all of them*, just the *first one*, so can you please open a new PR with the other changes you did and remove/rollback this one.